### ISSUE TITLE
Update recommended template priority for override in documentation to `500`

### ISSUE THREAD

**Issue by lahsivjar** [elastic/elasticsearch#115196]
### Description

APM team recently migrated to [APM data plugin](https://github.com/elastic/elasticsearch/tree/main/x-pack/plugin/apm-data) for managing our index templates, component templates, ingest pipelines, etc. As part of the change, we were forced to increase the priority of our index templates from `200` to `210` to override the fleet's priority. Fleet installs templates with a priority of either 150 or 200, documented to be anything between 100 and 200.

The current recommendation in https://www.elastic.co/guide/en/elasticsearch/reference/current/index-templates.html indicates to override with priority greater than `200` which won't work for APM cases now with our updated priority. I am proposing to increase the recommended priority number to `500` to give us enough room to make changes to managed templates when and if required.

There were similar changes made earlier to the doc in https://github.com/elastic/elasticsearch/issues/68982.

**Comment by endorama** [comment]
Reading through previous work here and our current docs, an additional point is that https://github.com/elastic/elasticsearch/issues/68982 already uses `500` as the value for all examples. Looking at the changes in https://github.com/elastic/elasticsearch/pull/69006, the PR that closes the mentioned issue, is possible to see that `500` has been used as value.

The only place where `200` is used is in this phrase:
> Assign templates with an overlapping pattern a priority higher than `200`. For example, if you don’t use Fleet or Elastic Agent and want to create a template for the logs-* index pattern, assign your template a priority of `500`. This ensures your template is applied instead of the built-in template for logs-*-*. 

@lahsivjar to confirm is what you expect to see `500` in place of `200` in the phrase above or am I missing something?

**Comment by lahsivjar** [comment]
> @lahsivjar to confirm is what you expect to see 500 in place of 200 in the phrase above or am I missing something?

Yeah. The examples use `500` but the actual statement suggests `200+`, so that needs to be updated.
